lib/publiccloud/instance.pm

```diff
@@ -976,4 +976,40 @@ sub wait_for_state {
     die("The instance state is not '$state' but '$current' instead.");
 }
 
+sub check_selinux_pc {
+    # Check SELinux-enforced VMs on publiccloud
+    my ($self) = @_;
+    my $log_avc = "selinux_avc_audit.txt";
+    my $ssh_opts = "-t -o ControlPath=none";
+    die("Unexpected environment.") unless (is_public_cloud && (is_sle_micro || is_sle));
+
+    my $instance = $self;    # $self->{my_instance};
+    my $out;
+
+    $out = $instance->ssh_script_output("cat /sys/kernel/security/lsm", ssh_opts => $ssh_opts);
```

Comment:
I think for a library function that checks if SELinux is enforcing, this is doing too much.

A library function should stick to it's own purpose, this is doing more than what the name suggests.